packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -86,10 +92,37 @@ class SparkComponent extends Component {
   }
 
   storeAndListen(sparkInstance) {
+    this.checkPlugins(sparkInstance);
     this.props.storeSparkInstance(sparkInstance);
     this.listenToSparkEvents(sparkInstance);
   }
 
+  checkPlugins(instance) {
+    for (const plugin of PLUGINS) {
+      // eslint-disable-next-line no-prototype-builtins
```

Comment:
Let's follow the suggestion from the [eslint error description](https://eslint.org/docs/rules/no-prototype-builtins) and use `Object.prototype.hasOwnProperty.call(instance, plugin);`.

Also, we've tried to prefix eslint disables with `// eslint-disable-reason my reason for disabling` in the code base.

Reply:
<img width="1264" alt="Screen Shot 2019-06-05 at 9 16 38 AM" src="https://user-images.githubusercontent.com/14226615/58992829-b56f5400-87a0-11e9-9864-f0f0218d4665.png">

Reply:
Please add this as an `// eslint-disable-reason` comment if you are not going to use the eslint preferred method.

Reply:
nvm. the reason does not apply to my case.